packages/network-controller/tests/NetworkController.test.ts

```diff
@@ -2159,103 +2122,11 @@ describe('NetworkController', () => {
-                        },
                         response: {
-                          result: '2',
+                          result: POST_1559_BLOCK,
                         },
                       },
-                    ]),
-                  ];
-                  const fakeNetworkClients = [
-                    buildFakeClient(fakeProviders[0]),
-                    buildFakeClient(fakeProviders[1]),
-                  ];
-                  mockCreateNetworkClient()
-                    .calledWith({
-                      network: networkType,
-                      infuraProjectId: 'some-infura-project-id',
-                      type: NetworkClientType.Infura,
-                      chainId: BUILT_IN_NETWORKS[networkType].chainId,
-                    })
-                    .mockReturnValue(fakeNetworkClients[0])
-                    .calledWith({
-                      chainId: toHex(1337),
-                      rpcUrl: 'https://mock-rpc-url',
-                      type: NetworkClientType.Custom,
-                    })
-                    .mockReturnValue(fakeNetworkClients[1]);
-                  await waitForStateChanges({
-                    messenger,
-                    propertyPath: ['networkId'],
-                    operation: async () => {
-                      await controller.initializeProvider();
-                    },
-                  });
-                  expect(controller.state.networkId).toBe('1');
-
-                  await waitForStateChanges({
-                    messenger,
-                    propertyPath: ['networkId'],
-                    operation: async () => {
-                      await controller.lookupNetwork();
-                    },
-                  });
-
-                  expect(controller.state.networkId).toBe('2');
-                },
-              );
-            });
-
-            it('stores the EIP-1559 support of the second network, not the first', async () => {
-              await withController(
-                {
-                  state: {
-                    providerConfig: buildProviderConfig({ type: networkType }),
-                    networkConfigurations: {
-                      testNetworkConfigurationId: {
-                        id: 'testNetworkConfigurationId',
-                        rpcUrl: 'https://mock-rpc-url',
-                        chainId: toHex(1337),
-                        ticker: 'ABC',
-                      },
-                    },
-                  },
-                  infuraProjectId: 'some-infura-project-id',
-                },
-                async ({ controller, messenger }) => {
-                  const fakeProviders = [
-                    buildFakeProvider([
-                      // Called during provider initialization
+                      // Called via `lookupNetwork` directly
                       {
                         request: {
                           method: 'net_version',
```

Comment:
looks like we're still mocking a bunch of `net_version` calls and responses in here? I assume we want to remove them?

Reply:
These tests check to see if rpc requests are actually made to the RPC endpoint or intercepted by createScaffoldMiddleware depending on if the network is built-in or custom.

Reply:
Hmmm ok, but it looks like you removed the net_version mock requests?

Reply:
ahh you're right, I meant to comment about the shared-test.ts scenario. This has been removed here like you said as NetworkController does not make a `net_version` request own its own anymore